protocols/dinosaur/dinosaur.ot2.apiv2.py

```diff
@@ -0,0 +1,44 @@
+metadata = {
+    'protocolName': 'Dinosaur',
+    'author': 'Opentrons <protocols@opentrons.com>',
+    'description': 'Draw a picture of a dinosaur',
+    'apiLevel': '2.9'
+}
+
+
+def run(ctx):
+
+    [p300_mount, tip_type, plate_type] = get_values(  # noqa: F821
+        "p300_mount", "tip_type", "plate_type")
+
+    # Load Labware
+    tiprack = ctx.load_labware(tip_type, 6)
+    plate = ctx.load_labware(plate_type, 3)
+    reservoir = ctx.load_labware('nest_12_reservoir_15ml', 8)
+
+    # Load Pipette
+    p300 = ctx.load_instrument('p300_single_gen2', p300_mount,
+                               tip_racks=[tiprack])
+
+    # Solutions
+    green = reservoir['A1']
+    blue = reservoir['A2']
+
+    # Wells to dispense green
+    green_wells = [well.bottom() for well in plate.wells(
```

Comment:
I think we should remove the `.bottom()` or add some height (1 or 2) to it.

Reply:
I removed `.bottom()` since I think by default it will aspirate/dispense 1 mm above the bottom.